### PR TITLE
#184759908 Updating and deleting product

### PR DESCRIPTION
**Pull request by fabmukunzi**
### What does this PR do?
Updating Product and Deleting product

### Description of Tasks to be completed?
A seller should be able to update and delete products   from their collection
### How should this be manually tested?
- Clone [repo](https://github.com/atlp-rwanda/vikings-ec-fe.git)
- Change into the directory **vikings-ec-fe**
- Checkout to the branch: `feature/update-product-184759908`
- Run the command `npm install` 
- Run the command `npm run dev`
- Try login as admin **user{email:admin@gmail.com,password:Password@123}**
-  Go to the route **/dashboard/products**
- Click on delete icon to delete
- Click on update icon to update

### What are the relevant pivotal tracker stories?
[#184759908](https://www.pivotaltracker.com/story/show/184759908)
[#184759909](https://www.pivotaltracker.com/story/show/184759909)

### CODE REVIEW

**Review by irakozetony — APPROVED** [review]

**Review by Munezeroolivierhugue — APPROVED** [review]

**Review by ndatumuremyi — APPROVED** [review]

**Comment by ncutixavier** [comment]
Remove all unnecessary file, starting from the icon because they are not belong in your task

File: src/components/forms/updateProduct.js [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+  } = useForm({
+    resolver: yupResolver(productSchema),
+  });
+  const { id } = useParams();
+  const today = new Date().toISOString().substr(0, 10);
+  const date = new Date(data?.expiryDate);
+  const year = date.getFullYear();
+  const month = String(date.getMonth() + 1).padStart(2, '0');
+  const day = String(date.getDate()).padStart(2, '0');
+  const extractedDate = `${year}-${month}-${day}`;
```
Comment (ncutixavier) [inline src/components/forms/updateProduct.js]: Create a util function to handle this functionality. You can get the format you need with `toISOString()`

File: src/pages/product/updateProduct.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import UpdateProductForm from '../../components/forms/updateProduct';
+
+const UpdateProductPage = () => {
+  return (
+    <div className="bg-[#f2edf3] max-h-[100%] max-w-[100%] p-[5%]">
+      <UpdateProductForm />
+    </div>
+  );
+};
+export default UpdateProductPage;
```
Comment (ncutixavier) [inline src/pages/product/updateProduct.js]: Let's have one file combining this file and the form

**Review by ncutixavier — CHANGES_REQUESTED** [review]

File: src/components/forms/updateProduct.js
Comment (ncutixavier) [inline src/components/forms/updateProduct.js:1]: This is product related component, not form

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**